src/Resource/Shipment.php

```diff
@@ -161,6 +91,14 @@ public function getInvoiceReference(): ?string
         return $this->invoiceReference;
     }
 
+    /**
+     * @param string|null $invoiceReference
+     */
+    public function setInvoiceReference(?string $invoiceReference): void
+    {
+        $this->invoiceReference = $invoiceReference;
+    }
```

Comment:
niet in de invoice, maar ik denk wel dat we altijd een externe referentie verplicht moeten stellen. Zodat we dit zichtbaar kunnen maken.

Reply:
In de externe koppelingen en integraties gebruiken we wel de invoiceReference om hun eigen referentie mee te geven.